src/ol_infrastructure/applications/ocw_site/snippets/ttl_setup.vcl

```diff
@@ -2,9 +2,17 @@ if (beresp.status == 404) {
   set beresp.ttl = 5m;
   set beresp.http.Cache-Control = "max-age=300";
   return(deliver);
- }
+}
 
-if (bereq.url.path ~ "^/main\.[0-9a-f]+\.(css|js)") {
+if (bereq.url.path ~ ".*(main|common|www|course_v2|instructor_insights|fields|[0-9a-f]+)\.[0-9a-f]+\.(css|js)") {
```

Comment:
Is there supposed to be a | "or" between `fields` and the hash match, `[0-9a-f]+)\.` ?

Why do we have the hash match twice? 

I know this is out of scope, but is there any reason for the parentheses in `(css|js)` -- aren't square brackets sufficient?